### PR TITLE
HPCC-12879 Add ESP Service URLs

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-12879

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

File: esp/src/eclwatch/TopologyDetailsWidget.js [hunk 1]
```diff
@@ -152,6 +152,31 @@ define([
...
+                    tpBinding = this.params.__hpcc_parentNode.__hpcc_treeItem;
+                };
+                if (tpBinding && tpMachine) {
+                    var tr = domConstruct.create("tr", {}, table);
+                    domConstruct.create("td", {
+                        innerHTML: "<b>URL:&nbsp;&nbsp;</b>"
+                    }, tr);
+                    var td = domConstruct.create("td", {
+                    }, tr);
+                    var url = tpBinding.Protocol + "://" + tpMachine.Netaddress + ":" + tpBinding.Port + "/";
```
Comment (miguelvazq) [inline esp/src/eclwatch/TopologyDetailsWidget.js]: is last "/" necessary?

Comment (GordonSmith) [inline esp/src/eclwatch/TopologyDetailsWidget.js]: Probably not - but doesn't hurt either

**Comment by miguelvazq** [comment]
Fine with me. Looks good.

**Pull request merged**